modules/porous_flow/doc/content/modules/porous_flow/tests/capillary_pressure/capillary_pressure_tests.md

```diff
@@ -1,3 +1,35 @@
 # Capillary pressure test descriptions
 
-TODO
+#### The detail documentation of the implementation of this code packet can be founded in [here](porous_flow/capillary_pressure.md).
+
+## Brooks-Corey
+
+!media capillary_pressure/brook_corey.png style=width:100%;margin-left:10px; caption=Brooks-Corey Capillary Pressure test cases id=brookcoreyPc
+
+The input file for log-extension off test:
+
+!listing modules/porous_flow/test/tests/capillary_pressure/brooks_corey1.i
+
+The input file for log-extension on test:
+
+!listing modules/porous_flow/test/tests/capillary_pressure/brooks_corey2.i
+
+
+
+## van Genuchten
+
+
+
+!media capillary_pressure/vangen.png style=width:100%;margin-left:10px; caption=van Genuchten Capillary Pressure test cases id=vangenPc
+
```

Comment:
Maybe extend the x range to include 0 to show what happens? If the y axis gets too large, you could use a log scaling on that axis, or just limit the yrange

Reply:
Also a typo in the plot heading - can you carefully check these?

Reply:
Can you have a look on this figure again for the analytical curve to see if it is correct?